src/components/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Link, useLocation } from 'react-router-dom';
+
+const Navbar: React.FC = () => {
+  const location = useLocation();
+
+  const isActive = (path: string) =>
+    location.pathname === path ||
+    (path === '/people' && location.pathname.startsWith('/people'));
```

Comment:
You can add all routes in the enum and use it in all places in your application.